### PR TITLE
Update CSES Labyrinth.mdx

### PR DESCRIPTION
**Pull request by TruaShamu**
add java implementation to cses labyrinth

### CODE REVIEW

**Comment by TruaShamu** [comment]
...Since it's passed all the checks, can we merge the pr?

File: solutions/CSES Labyrinth.mdx [hunk 1]
```diff
@@ -98,3 +101,112 @@ int main() {
 	return 0;
 }
 ```
+</CPPSection>
+
+<JavaSection>
+
+```java
```
Comment (maggie-j-liu) [inline solutions/CSES Labyrinth.mdx:108]: use tabs instead of spaces to indent

File: solutions/CSES Labyrinth.mdx [hunk 2]
```diff
@@ -98,3 +101,112 @@ int main() {
...
+import java.io.*;
+import java.util.*;
+
+public class cses1193 {
+    public static int[] dX = {-1, 0, 0, 1};
+    public static int[] dY = {0, -1, 1, 0};
+    public static String dirs = "ULRD";
+
+    // Coordinates for point A and B.
```
Comment (maggie-j-liu) [inline solutions/CSES Labyrinth.mdx:117]:
```suggestion
    //Coordinates for points A and B.
```

**Review by maggie-j-liu — CHANGES_REQUESTED** [review]

**Comment by maggie-j-liu** [comment]
Usually you need 1-2 team approvals and 1 core approval to merge; you can request reviews since you're a content author.

**Comment by TruaShamu** [comment]
ok, thanks.

**Review by dongliu0426 — APPROVED** [review]

**Review by maggie-j-liu — APPROVED** [review]

**Pull request merged**